src/com/gfredericks/test/chuck/clojure_test.cljc

```diff
@@ -0,0 +1,81 @@
+(ns com.gfredericks.test.chuck.clojure-test
+  (:require [clojure.test.check :as tc]
+            [clojure.test.check.properties :as prop
+             #?@(:cljs [:include-macros true])]
+            [com.gfredericks.test.chuck.clojure-test.impl
+             :refer [pass? report-exception save-to-final-reports]]))
+
+(def ^:dynamic *chuck-captured-reports*)
+
+#?(:cljs
+(defmethod cljs.test/report [::chuck-capture :fail]
+  [m]
+  (swap! *chuck-captured-reports* conj m)))
+
+#?(:cljs
+(defmethod cljs.test/report [::chuck-capture :pass]
+  [m]
+  (swap! *chuck-captured-reports* conj m)))
+
+(defn- cljs-env?
+  "Take the &env from a macro, and tell whether we are expanding into cljs."
+  [env]
+  (boolean (:ns env)))
+
+(defmacro if-cljs
+  "Return then if we are generating cljs code and else for Clojure code.
+   https://groups.google.com/d/msg/clojurescript/iBY5HaQda4A/w1lAQi9_AwsJ"
+  [then else]
+  (if (cljs-env? &env) then else))
+
+(defmacro capture-reports [& body]
+  `(let [reports# (atom [])]
+     (if-cljs
+       (binding [*chuck-captured-reports* reports#
+                 cljs.test/*current-env* (cljs.test/empty-env ::chuck-capture)]
+         ~@body)
+       (binding [clojure.test/report #(swap! reports# conj %)]
+         ~@body))
+     @reports#))
```

Comment:
Yessss! of course we can, thanks :)

Done in bce3f46c67d0edb8ed281561e2a96fd9d44dae0f